primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:13:03.35,0:13:06.46,Default,edit,0,0,0,,You don't have an insubordinate \Nbone in your body, do you?{You're as dutiful as they come, aren't you?}{like the impulse to question his orders would never enter her mind}
 Dialogue: 0,0:13:07.01,0:13:07.89,Default,,0,0,0,,What's that?
 Dialogue: 0,0:13:19.44,0:13:21.78,Default,,0,0,0,,Rin, focus!
-Dialogue: 0,0:13:25.38,0:13:27.04,Default,,0,0,0,,I'm fine, I'm fine.
+Dialogue: 0,0:13:25.38,0:13:27.04,Default,,0,0,0,,I'm good! I'm good.
 Dialogue: 0,0:13:27.04,0:13:28.15,Default,,0,0,0,,I can still fight!
-Dialogue: 0,0:13:28.15,0:13:30.40,Default,,0,0,0,,Rin... no.
+Dialogue: 0,0:13:28.15,0:13:30.40,Default,,0,0,0,,Rin—No.
 Dialogue: 0,0:13:30.40,0:13:31.68,Default,,0,0,0,,All units, retreat.
 Dialogue: 0,0:13:33.16,0:13:35.17,Default,,0,0,0,,I'll finish it off.
-Dialogue: 0,0:13:35.17,0:13:38.77,Default,,0,0,0,,As long as we know its weakness,\NI can simply charge into it with my unit!{弱点さえ分かれば、こいつで突っ込むのもわけないさ}{pretty sure he means as a suicide attack, with the alternative in his mind being one of the girls doing the same thing}
-Dialogue: 0,0:13:38.77,0:13:40.73,Default,,0,0,0,,Funagai! Have you gone insane?
-Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself it was just work.
-Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I was able to cope by convincing myself it was professional.
-Dialogue: 0,0:13:49.37,0:13:50.31,Default,,0,0,0,,But...
-Dialogue: 0,0:13:51.44,0:13:52.71,Default,,0,0,0,,those girls
-Dialogue: 0,0:13:53.21,0:13:54.40,Default,,0,0,0,,were smiling.
-Dialogue: 0,0:13:56.11,0:13:58.30,Default,,0,0,0,,That's why... That's why...
+Dialogue: 0,0:13:35.17,0:13:38.77,Default,,0,0,0,,We know its weak spot now, \Nso all I've gotta do is ram into it with my unit!{弱点さえ分かれば、こいつで突っ込むのもわけないさ}{pretty sure he means as a suicide attack, with the alternative in his mind being one of the girls doing the same thing}
+Dialogue: 0,0:13:38.77,0:13:40.73,Default,,0,0,0,,Funagai! Have you lost your mind?
+Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself I was just doing my job.
+Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by convincing \Nmyself that was true.
+Dialogue: 0,0:13:49.37,0:13:52.71,Default,,0,0,0,,But thinking back on it...
+Dialogue: 0,0:13:53.21,0:13:54.40,Default,,0,0,0,,all I can see is their smiles.
+Dialogue: 0,0:13:56.11,0:13:58.30,Default,,0,0,0,,So, I... I—{Na: For some reason I get the feeling he's trying to say that he loves these robochicks, so making this a little more vague.}
 Dialogue: 0,0:14:13.50,0:14:15.93,Default,,0,0,0,,Rin! What are you doing?
-Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I didn't follow your orders all this time because of how I was created!
-Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,It was because I loved you!
+Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I haven't followed your orders just \Nbecause I was created to do so!
```

Comment:
Awkward line break. I would also use the past continuous here.
```suggestion
Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I haven't been following your orders \Njust because I was created to do so!
```
Alternatively, to stick closer to the original TL:
```suggestion
Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I wasn't following your orders \Njust because I was created to do so!
```